### PR TITLE
TWC2018 first teams update

### PR DESCRIPTION
**Pull request by juankristal**
reeeeeeeeeeeeeeeeeeeeee

### CODE REVIEW

File: wiki/Tournaments/TWC/2018/en.md [hunk 1]
```diff
@@ -29,33 +29,68 @@ In every world cup, people conquering a place on the podium are eligible to rece
 
 | Placing | Prize(s) |
 | --- | :--- |
-| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | profile badge, "osu!taiko Champion" user title |
-| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | profile badge |
-| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | profile badge |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $150 per team member, profile badge, "osu!mania Champion" user title for one year |
```
Comment (MegaApplePi) [inline wiki/Tournaments/TWC/2018/en.md]: I thought this was TWC, not MWC

**Comment by MegaApplePi** [comment]
5 (more or less) flags broke because their reference links were removed

File: wiki/Tournaments/TWC/2018/en.md [hunk 2]
```diff
@@ -29,33 +29,68 @@ In every world cup, people conquering a place on the podium are eligible to rece
...
+| ![][flag_MY] | **Malaysia** | **[Xeltic Rival](https://osu.ppy.sh/users/7500364)** |
+| ![][flag_RU] | **Russian Federation** | **[Alt](https://osu.ppy.sh/users/736496)** |
+| ![][flag_ES] | **Spain** | **[Raiden](https://osu.ppy.sh/users/2239480)** |
+| ![][flag_TW] | **Taiwan** | **[-[ ix Ishida xi ]-](https://osu.ppy.sh/users/242910)** |
+| ![][flag_GB] | **United Kingdom** | **[goheegy](https://osu.ppy.sh/users/8057655)** |
+| ![][flag_US] | **United States** | **[n1doking](https://osu.ppy.sh/users/5991961)** |
+
+
+
+_Remember this is not the full list of team, some of them are still missing_
```
[the remark this answers is not in the document]
Comment (juankristal) [reply wiki/Tournaments/TWC/2018/en.md]: Changed the teams thing, the other one I reckon is alright as it is.

File: wiki/Tournaments/TWC/2018/en.md [hunk 3]
```diff
@@ -29,33 +29,68 @@ In every world cup, people conquering a place on the podium are eligible to rece
...
-| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | profile badge |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | $150 per team member, profile badge, "osu!taiko Champion" user title for one year |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | $80 per team member, profile badge |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | $40 per team member, profile badge |
 
-_More prizes to be defined_
 
 ## Organization
 
 The osu!taiko World Cup 2018 is ran by various community members by distributing the multitude of tasks into various fields of responsibility.
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: Is run

File: wiki/Tournaments/TWC/2018/en.md [hunk 4]
```diff
@@ -29,33 +29,68 @@ In every world cup, people conquering a place on the podium are eligible to rece
```
Comment (TPGPL) [inline wiki/Tournaments/TWC/2018/en.md]: Btw change the Drawings hour to 14:00 instead

**Pull request merged**